### DISCUSSION TITLE
In need of a beginner's guide

### DISCUSSION THREAD

**Discussion by ssiyad** [nvim-neorg/neorg#847] [Q&A] [answered]
Hello, I'm new to Neorg and org-mode. Is there a beginner's guide (with nice defaults) that I can follow? I need completion, TreeSitter integration, and navigation between days/weeks. I checked the README and wiki, but they do not seem like a guide (rather a spec).

**Comment by d-r-a-b** [comment, marked as answer]
If you are looking for org-mode, you probably want to be looking in https://orgmode.org/ for Emacs or https://github.com/nvim-orgmode/orgmode. These are more mature projects with their own momentum, tooling and established workflows.

For your requirements, like a planner with navigation between days/week, this simply doesn't exist yet in the Neorg platform *yet*.

Neorg is a new project based on a different file format, recently specified in https://github.com/nvim-orgmode/orgmode. To understand more, see https://github.com/nvim-neorg/neorg#-philosophy. As a project, it has an extremely promising set of features that are possible because of the decisions made in designing the `.norg` file format, but many of those more impressive applications are *in potentia* even if they are rapidly being developed and exciting things are happening week-to-week behind the scenes. See https://www.youtube.com/watch?v=NnmRVY22Lq8&list=PLx2ksyallYzVI8CN1JMXhEf62j2AijeDa for a youtube playlist "From No Org to Neorg" from the project lead to get a friendly intro. See also https://github.com/pysan3/Norg-Tutorial for some setup information.

At this stage, Neorg's best feature is it's very easily parse-able `.norg` format and it also has a nice event-based module mechanism. This makes it ideal for any dev that wants to build out customizable functionality. Links are obviously a great feature, but most rich-text formats have these and there are many, many plugins which allow you to quickly jump from one file to another (or use vim's own `gf`). It also has very nice user-facing features that can be configured like the concealer and TODO keybinds, automatic metadata injection + update, ability to interleave code blocks and tangle them into an output file, etc.

**Comment by ssiyad** [comment]
Thank you @d-r-a-b for the detailed reply!
